src/views/v2/components/loader/loader.php

```diff
@@ -14,14 +14,18 @@
  * @version 5.0.4
  */
 
-$classes = $this->get( 'classes' ) ?: [];
+$classes        = $this->get( 'classes' ) ?: [];
+$visible        = tribe_is_truthy( $this->get( 'visible' ) ) || false;
 
 $spinner_classes = [
 	'tribe-tickets-loader__dots',
 	'tribe-common-c-loader',
-	'tribe-common-a11y-hidden',
 ];
 
+if ( ! $visible ) {
+	$spinner_classes[] = 'tribe-common-a11y-hidden';
```

Comment:
We could avoid the if by doing something like this:

```
$spinner_classes = [
	'tribe-common-a11y-hidden => empty( $visible ),
]
```

(as part of the definition of the `$spinner_classes` above).